cli/azd/internal/repository/app_init.go

```diff
@@ -1107,12 +1112,61 @@ func promptSpringBootVersion(console input.Console, ctx context.Context) (string
 	}
 }
 
+func promptMissingEventHubsNameOrExit(console input.Console, ctx context.Context, eventHubs *appdetect.AzureDepEventHubs) {
+	switch eventHubs.FromDependency {
```

Comment:
Is it possible that multiple `FromDependency` exists at the same time?

Reply:
It's impossible because we specify one `FromDependency` when constructing event hubs. Even if there are multiple dependencies related to event hubs, it should have multiple event hubs here, instead of multiple `FromDependency`.

Reply:
In current spec.go:
1. Only support one event hub service:
> ![image](https://github.com/user-attachments/assets/d3a0aa7c-7201-4a06-99a6-b3f06ed658b3)

2. One event hub service can contains multiple event hub names:
> ![image](https://github.com/user-attachments/assets/eda40693-2c61-47c5-8e43-b5ef3aeeafc3)

So is it possible that there are multiple `FromDependency`s, and one `FromDependency` map to one/multiple event hub name?

Reply:
Thanks. Considering that the `Project` has an array of `AzureDep` but `InfraSpec` has the individual Azure Dep declaration, we have to make **only one Azure Event Hubs and possibly multiple From Dependencies**. 
Updated code to make it.